lib/main_common.pm

```diff
@@ -1231,7 +1231,7 @@ sub load_x11tests {
         loadtest "x11/hexchat";
         loadtest "x11/vlc";
     }
-    if (gnomestep_is_applicable() && !is_staging()) {
+    if (is_sle && gnomestep_is_applicable() && !is_staging()) {
```

Comment:
Please mention a progress ticket above which you track to enable it again

Reply:
Done, please check now.